iothub/device/tests/IotHubDeviceClientTests.cs

```diff
@@ -119,25 +119,25 @@ public void DeviceAuthenticationWithX509Certificate_ChainCertsMqtttTcp_DoesNotTh
             var options = new IotHubClientOptions(new IotHubClientMqttSettings(IotHubClientTransportProtocol.Tcp));
 
             // act
-            using var dc = new IotHubDeviceClient(hostName, authMethod, options);
+            await using var dc = new IotHubDeviceClient(hostName, authMethod, options);
```

Comment:
Another case here -- we use `dc` and `deviceClient` as variable names in the same file to mean the same type. It would be cleaner if we had a consistent name, imho